lib/web_ui/dev/test_runner.dart

```diff
@@ -180,14 +178,55 @@ class TestCommand extends Command<bool> with ArgUtils {
       });
     }
 
+    await _buildTargets();
+
     if (runAllTests) {
-      await _runAllTests();
+      await _runAllTestsForCurrentPlatform();
     } else {
-      await _runTargetTests(targetFiles);
+      await _runSpecificTests(targetFiles);
```

Comment:
question: why do we rename?

Reply:
Because `_runAllTests` also runs test targets, so I though there was insufficient distinction. "Specific" really implies that someone _specified_ the exact tests to run.